tests/test/snapshot/test_snapshot_diffs.cpp

```diff
@@ -36,9 +36,13 @@ TEST_CASE_METHOD(SnapshotTestFixture,
 
     int sharedMemPages = 8;
     size_t sharedMemSize = sharedMemPages * HOST_PAGE_SIZE;
-    MemoryRegion sharedMem = allocateSharedMemory(sharedMemSize);
+    MemoryRegion sharedMem = allocatePrivateMemory(sharedMemSize);
```

Comment:
Just to double check, the variable is called `sharedMem` and is defined by calling `allocatePrivateMemory` which feels odd.